packages/app-store/googlecalendar/zod.ts

```diff
@@ -5,5 +5,11 @@ export const appDataSchema = z.object({});
 export const appKeysSchema = z.object({
   client_id: z.string().min(1),
   client_secret: z.string().min(1),
-  redirect_uris: z.string().min(1),
+  redirect_uris: z.union([
+    z.string().url().array(),
+    z
+      .string()
+      .url()
+      .transform((url) => [url]),
```

Comment:
When submitting keys, we parse the input so it would automatically transform the `redirect_uris` to an array. So we either accept an array or a single string and transform it into an array.